index.bs

```diff
@@ -8041,38 +8043,32 @@ ECMAScript Object values.
 
 <h4 id="es-promise">Promise types — Promise&lt;|T|&gt;</h4>
 
-IDL [=promise type=] values are
-represented by ECMAScript {{ECMAScript/Promise}}
-objects.
+IDL [=promise type=] values are represented by ECMAScript {{ECMAScript/PromiseCapability}} records.
```

Comment:
Nit: This is not an interface, so shouldn't be using IDL syntax or be tagged as such in the Bikeshed metadata.